repose-aggregator/components/filters/header-normalization-filter/src/test/scala/org/openrepose/filters/headernormalization/HeaderNormalizationFilterTest.scala

```diff
@@ -48,21 +49,36 @@ class HeaderNormalizationFilterTest extends FunSpec with BeforeAndAfterEach with
   var filter: HeaderNormalizationFilter = _
   var servletRequest: MockHttpServletRequest = _
   var servletResponse: MockHttpServletResponse = _
-  var filterChain: FilterChain = _
-  var filterConfig: FilterConfig = _
+  private val filterChain = mock[FilterChain]
+  private val filterConfig = mock[FilterConfig]
+  private val JMX_PREFIX = "MOCK-PREFIX-"
+  private val METRIC_CLASS = "org.openrepose.core.filters.HeaderNormalization"
+  private val TYPE = "header-normalization"
+  private val SCOPE = "Normalization"
+  private val METRIC_PREFIX = s"$JMX_PREFIX$METRIC_CLASS.$TYPE.$SCOPE."
   var metricsService: MetricsService = _
-  var metricsMeters: MeterByCategorySum = _
+  private val metricRegistry = mock[MetricRegistry]
+  private val meter = mock[Meter]
 
   override def beforeEach(): Unit = {
     servletRequest = new MockHttpServletRequest
     servletResponse = new MockHttpServletResponse
-    filterChain = mock[FilterChain]
-    filterConfig = mock[FilterConfig]
-    metricsService = mock[MetricsService]
-    metricsMeters = mock[MeterByCategorySum]
+
+    reset(filterChain)
+    reset(filterConfig)
+    reset(metricRegistry)
+    reset(meter)
 
     when(filterConfig.getInitParameterNames).thenReturn(List.empty[String].toIterator.asJavaEnumeration)
-    when(metricsService.newMeterByCategorySum(any(), any(), any(), any())).thenReturn(metricsMeters)
+    when(metricRegistry.meter(anyString())).thenReturn(meter)
+
+    metricsService = new MetricsService {
+      override def getRegistry: MetricRegistry = metricRegistry
+
+      override def name(className: String, names: String*): String = MetricRegistry.name(JMX_PREFIX + className, names:_*)
```

Comment:
I just wanted to point out that this is the Scala syntax, `names:_*`, to pass an incoming Java varargs along to another Java method.